### PR TITLE
Updates to bankruptcy courts for Hawaii, Nebraska, and Texas.

### PR DESCRIPTION
**Pull request by jonashleyo**
Changed id for United States Bankruptcy Court, D. Nebraska from "nebraskab" to "neb".
Removed "Middle" from "United States Bankruptcy Court For The Middle District Of Hawaii".
Added Bankruptcy Court for the Western District of Texas.  Running a search without this returns results for 'txwb' and 'txeb'.

### CODE REVIEW

File: courts_db/data/courts.json [hunk 1]
```diff
@@ -6289,7 +6289,7 @@
         "examples": [],
         "court_url": "http://www.neb.uscourts.gov/",
         "type": "bankruptcy",
-        "id": "nebraskab",
```
Comment (flooie) [inline courts_db/data/courts.json:6292]: Unfortunately, your change in Nebraska makes sense, but we already have nebraskab - in our CL system.  I don't think the plan would be to change the court codes already in the system.

**Comment by brianwc** [comment]
Yeah, because the state court is 'Neb' it is one of the few unfortunate
collisions that requires us to use an id different from the subdomain that
precedes 'uscourts.gov'.

**Comment by mlissner** [comment]
I usually the reviewer on things by default, but @flooie is the master here, so I'm going to let him review, merge, etc this one and I'll steer clear unless I hear otherwise.

File: courts_db/data/courts.json (hunk 1 above)
Comment (jonashleyo) [reply courts_db/data/courts.json:6292]: Makes sense.  I'm still relatively green with this process but should I resubmit this pull request with only the changes for Hawaii and Texas?

Comment (mlissner) [reply courts_db/data/courts.json:6292]: The easiest thing is to make the change, add another commit, and push it to this branch. That'll update this PR.

File: courts_db/data/courts.json [hunk 2]
```diff
@@ -3537,7 +3537,7 @@
         "regex": [
             "(^|\\s)D(\\.|(istrict))? ?Hawai",
             "${usbcd} Hawai",
-            "United States Bankruptcy Court For The Middle District Of Hawaii"
+            "United States Bankruptcy Court For The District Of Hawaii"
```
Comment (flooie) [inline courts_db/data/courts.json:3540]: Great find.

**Pull request merged**